pkg/prowloader/github/github.go

```diff
@@ -143,20 +173,88 @@ func (c *Client) fetchPR(prl prlocator) (*prentry, error) {
 		return nil, err
 	}
 
-	var state *prentry
+	c.cache[prl] = pr
+	return pr, nil
+}
+
+// This is an uncached call to github to get the most up to date information
+// on the PR.  Use cautiously and only when necessary
+func (c *Client) PRFetch(org, repo string, number int) (prEntry *PREntry, err error) {
+	// Get PR from GitHub
```

Comment:
@stbenjam Let me know if there are concerns with adding this here.  I do want to make sure I have the most recent copy before adding a comment but if putting this functionality in a new struct altogether with the other commenting methods.